api/src/accounts.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::context::Context;
+
+use diem_api_types::{Address, Error, MoveResource};
+use diem_types::account_state::AccountState;
+use resource_viewer::MoveValueAnnotator;
+
+use anyhow::Result;
+use std::{convert::TryFrom, str::FromStr};
+use warp::{Filter, Rejection, Reply};
+
+pub fn routes(context: Context) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    get_account_resources(context)
+}
+
+// GET /accounts/<address>/resources
+pub fn get_account_resources(
+    context: Context,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    warp::path!("accounts" / String / "resources")
+        .and(warp::get())
+        .and(context.filter())
+        .and_then(handle_get_account_resources)
+}
```

Comment:
This seems to be copy-pasted elsewhere.

Reply:
there are 3 commits, one of them re-organized code structure.
before change: grouped filters together, grouped handlers together, and grouped tests together.
after change: for one resource, like accounts, group it's filters, handlers and tests together.

This way, when adding new resource, we add one new mod for all of filters, handlers and tests together.

Reply:
just found I didn't delete filters.rs and handlers.rs after moved related functions into index.rs.
it is resolved now.